packages/client/Usage.md

```diff
@@ -38,8 +38,8 @@ To get a collection of records, you can use the `Query` object. It provides the
 
 - `getFirst()`: returns the first record in the query results.
 - `getPaginated()`: returns a page of records in the query results.
-- `getAll()`: returns all the records in the query results.
-- `getMany()`: returns an array of some records in the query results.
+- `getAll()`: returns all the records in the query results. **We recommend avoiding this method and using `getMany` or `getPaginated` instead, as large datasets can cause performance problems in your applications.**
+- `getMany()`: returns 20 results by default (for more defaults, see [helper variables](#helper-variables)), but is customizable via [pagination](#page).
```

Comment:
```suggestion
- `getMany()`: returns an array with a subset of the first results in the query. The default pagination size (20) is used and can be customised by passing a different `{ pagination: { size: number } }` in its options.
```

Reply:
This is great. Let's still link to helper variables for more info?

```suggestion
- `getMany()`: returns an array with a subset of the first results in the query. The default [pagination](#page) size (20) is used and can be customised by passing a different `{ pagination: { size: number } }` in its options. To learn more about default sizes, see [helper variables](#helper-variables).
```